### DISCUSSION TITLE
Warning when comparing tasks between GEMs using compareMetabolicModels

### DISCUSSION THREAD

**Discussion by CKTJames** [SysBioChalmers/Human-GEM#835] [Q&A] [answered]
Hi!

I've been following the Human_GEM guide to run ftINIT, and have managed to run ftINIT on the provided GTEx data. I also wanted to trial the "GEM comparison" part of the tutorial with the models generated from running the ftINIT on the GTex data (rather than the provided GEMs from the Human-GEM publication). However, when comparing metabolic tasks between GEMs I got the following warning: 
WARNING: Exchange metabolites should normally not be removed from the model when using checkTasks. Inputs and outputs are defined in the task file instead. Use importModel(file,false) to import a model with exchange metabolites remaining

Additionally, there were no differences between GEMs in metabolic task ability. 

Is this due to an error in how I carried out the ftINIT? If so, how can I troubleshoot the problem?

Thanks in advance!

**Comment by JonathanRob** [comment, marked as answer]
Hi @CKTJames! I can't remember if ftINIT outputs the GEM in closed or open form (with or without boundary metabolites, respectively), but you can try to convert each of the GEMs being compared to closed form using the `addBoundaryMets` function before running the comparison analyses.
